pkg/controller/rmdworkload/rmdworkload_controller.go

```diff
@@ -274,6 +248,50 @@ func (r *ReconcileRmdWorkload) findTargetedNodes(request reconcile.Request, rmdN
 	return targetedNodes, nil
 }
 
+func (r *ReconcileRmdWorkload) findRemovedNodes(request reconcile.Request, rmdNodeStates *intelv1alpha1.RmdNodeStateList, rmdWorkload *intelv1alpha1.RmdWorkload) (map[string]string, error) {
+	// Perform final check on RMD workloads vs rmdWorkload.Spec.Nodes to find any
+	// Nodes which may have been removed from the rmdWorkload since last reconcile.
+	// In this event, append node address and workload address to a map which will be returned.
```

Comment:
I think we can remove this. Instead, we should have a comment above this function that says 
`````
// findRemovedNodes finds Nodes that have the reconciled workload actively running, but those Nodes have been 
// removed from the RmdWorkload spec. Such instances are returned as a map of address (of RMD Pod) to workload
// ID so that the workload can be deleted from RMD.